ingestion/functions/parsing/brazil_distrito_federal/dictionaries.json

```diff
@@ -0,0 +1,143 @@
+{
+    "comorbidities": {
+        "Pneumopatia": "lung disease",
+        "Nefropatia": "kidney disease",
+        "Distúrbios Metabólicos": "disease of metabolism",
+        "Obesidade": "obesity",
+        "Cardiovasculopatia": "cardiovascular system disease"
+    },
+    "admin_districts": {
+        "Plano Piloto": {
+            "latitude": -15.7939,
+            "longitude": -47.8828
+        },
```

Comment:
I'm OK with these being here but wanted to ask what the motivation is, does map box get these wrong?

Reply:
Yeah exactly, see https://github.com/globaldothealth/list/issues/1407. essentially because the Distrito Federal in Brazil is not organized like other States but has it's own Administrative divisions, these are not in Mapbox and were geocoded wrong

Reply:
thanks!